README.md

```diff
@@ -214,7 +214,40 @@ sudo emerge --ask awscli
 ```
 
 ### azure
-TBD (FIXME)
+`azure` uses `~/.azure/azureProfile.json`. This can be created using the `az` [command](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli):
+```
+$ az login`
+```
+It also requires that the environment variable `AZURE_AUTH_LOCATION` points to a JSON file (this can also be set via the `--azure-auth` parameter). The JSON file will require a service provider active directory account to be created.
+
+Service provider accounts can be created via the `az` command (the output will contain an ID which is used as the `clientId` variable in the `AZURE_AUTH_LOCATION` JSON):
```

Comment:
Would be good to note that you want the `appId` field of the output here, since there are a few other ID-looking fields.